### PR TITLE
Add arm schema files for Shared image gallery service new API version

### PR DESCRIPTION
**Pull request by vanbasten2323**

### CODE REVIEW

File: schemas/2015-01-01/deploymentTemplate.json [hunk 1]
```diff
@@ -859,6 +859,9 @@
                   { "$ref": "https://schema.management.azure.com/schemas/2018-10-01/Microsoft.Compute.json#/resourceDefinitions/virtualMachineScaleSets" },
                   { "$ref": "https://schema.management.azure.com/schemas/2018-10-01/Microsoft.Compute.Extensions.json#/resourceDefinitions/virtualMachines_extensions" },
                   { "$ref": "https://schema.management.azure.com/schemas/2018-10-01/Microsoft.Compute.Extensions.json#/resourceDefinitions/virtualMachineScaleSets_extensions" },
+                  { "$ref": "https://schema.management.azure.com/schemas/2019-03-01/Microsoft.Compute.Galleries.json#/resourceDefinitions/galleries" },
```
Comment (TianoMS) [inline schemas/2015-01-01/deploymentTemplate.json]: >Microsoft.Compute.Galleries.json [](start = 84, length = 32)

PR looks good. one question: we don't have 2019-03-01 version of Microsoft.Compute.json today?

File: schemas/2015-01-01/deploymentTemplate.json [hunk 2]
```diff
@@ -859,6 +859,9 @@
...
+                  { "$ref": "https://schema.management.azure.com/schemas/2019-03-01/Microsoft.Compute.Galleries.json#/resourceDefinitions/galleries_images" },
+                  { "$ref": "https://schema.management.azure.com/schemas/2019-03-01/Microsoft.Compute.Galleries.json#/resourceDefinitions/galleries_images_versions" },
```
Comment (TianoMS) [inline schemas/2015-01-01/deploymentTemplate.json]: >                   [](start = 167, length = 18)

nit: remove extra spaces.

File: schemas/2015-01-01/deploymentTemplate.json (hunk 1 above)
Comment (vanbasten2323) [reply schemas/2015-01-01/deploymentTemplate.json]: It is the latest api version for azure compute. People are gradually adding support of this api version.

File: schemas/2015-01-01/deploymentTemplate.json (hunk 2 above)
Comment (vanbasten2323) [reply schemas/2015-01-01/deploymentTemplate.json]: Thanks. I've fixed it.

**Pull request merged**